packages/string-machine/PKGBUILD

```diff
@@ -22,7 +22,7 @@ sha256sums=('5b0d2eb2185199c1de6c7c35700a2caafbc6ba564529a2e7614c15c3aceacc6f'
 prepare() {
   cd "$pkgname-$pkgver"
   # Update DPF base makefile to fix arm64 build
-  cp -f "$srcdir"/string-machine-Makefile.base.mk dpf/Makefile.base.mk
+  cp -f "$srcdir"/${pkgname}-Makefile.base.mk dpf/Makefile.base.mk
```

Comment:
```suggestion
  cp -f "$srcdir"/$pkgname-Makefile.base.mk dpf/Makefile.base.mk
```

Reply:
Oh, did we agree on this? I though t you preferred the other style when a variable name is followed by a hyphen?